en/docs/administer/encrypting-passwords-with-cipher-tool.md

```diff
@@ -73,43 +71,15 @@ To change any password that we have encrypted already, follow the below steps:
 6. The system will then prompt you to select the alias of the password which you want to change. Enter the list number of the password alias.
 7. The system will then prompt you (twice) to enter the new password. Enter your new password.
 
-## Resolving encrypted passwords
-
-There are two ways to resolve encrypted passwords:
-
-!!! note 
-    If you have secured the plain text passwords in configuration files using Secure Vault, the keystore password and private key password of the product's [primary keystore](../../administer/configuring-keystores-in-wso2-products) will serve as the root passwords for Secure Vault. This is because the keystore passwords are needed to initialise the values encrypted by the **Secret Manager** in the **Secret Repository**. Therefore, the **Secret Callback handler** is used to resolve these passwords. The default secret CallbackHandler provides the two options given below. Read more about [Secure Vault concepts](../../administer/carbon-secure-vault-implementation).
-
-### Enter password in command-line
-1. Start the server by running the product start up script from the `<IS_HOME>/bin/` directory as shown below.
-   ```
-   ./server.sh
-   ```
-2. When you run the startup script, the following message will be prompted before starting the server: `[Enter KeyStore and Private Key Password:]`. This is because, in order to connect to the default user store, the encrypted passwords should be decrypted. The administrator starting the server must provide the private key and keystore passwords using the command-line. Note that passwords are hidden from the terminal and log files.
-
-### Start server as a background job
-
-If you start the Micro Integrator as a background job, you will not be able to provide password values on the command line. Therefore, you must start the server in "daemon" mode as explained below.
-
-1. Create a new file in the <IS_HOME> directory. The file should be named according to your OS as explained below.
-
-    * For Linux: The file name should be `password-tmp`.
-    * For Windows: The file name should be `password-tmp.txt`.
+## What's next?
 
-2. Add the primary keystore password (which is "wso2carbon" by default) to the new file and save. By default, the password provider assumes that both private key and keystore passwords are the same. If not, the private key password must be entered in the second line of the file.
+For information on resolving the encrypted passwords see [Resolving Encrypted Passwords](../../administer/resolving-encrypted-passwords).
 
-3. Now, start the server as a background process by running the following command.
-   ```
-   ./server.sh start
-   ```
-4. Start the server by running the product start-up script from the MI_HOME/bin directory by executing the following command:
-   ```
-   daemon. sh server.sh -start
-   ```
+For information on customizing secure vault implementaion see [Customizing Secure Vault](../../administer/customizing-secure-vault).
```

Comment:
Let's add this to an infobox as shown below:

!!! info
    For information on xxxx, see xxx .